src/index.ts

```diff
@@ -8,6 +8,8 @@
  * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
  */
 
+/// <reference path="../custom_typings/main.d.ts" />
```

Comment:
hmm... seems a bit unfortunate to expose our custom_typings. We should work on upstreaming.